### DISCUSSION TITLE
Is there a way to change several components' translation propagation settings at once?

### DISCUSSION THREAD

**Discussion by AlphaStarguide** [WeblateOrg/weblate#10065] [Q&A] [answered]
As title, I have nearly 70 components and I need to disable the translation propagation function for them all.
Is there any smarter way to do it rather than set it one by one?

**Comment by Geeyun-JY3** [comment]
#3166、https://github.com/WeblateOrg/weblate/discussions/10003#discussioncomment-7111857

**Reply by AlphaStarguide** [reply]
Thank you.
I saw I should disable the setting in component configuration, but I need a quicker way to do it on 70+ components.

**Reply by nijel** [reply, marked as answer]
Use API for that. There is a PATCH method on the component, see https://docs.weblate.org/en/latest/api.html#patch--api-components-(string-project)-(string-component)-
